string name
uint8[][] cropped_images

---

bool success